docker-entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
```

Comment:
I like to do `set -eux -o pipefail` to my scripts, but I don't really mind.

Reply:
Looks useful, done.